pkg/env/vuln_report_enhancements.go

```diff
@@ -0,0 +1,6 @@
+package env
```

Comment:
We already have `pkg/env/vuln_reporting.go` . Might be good to add the env var there to this file.

Reply:
I ack your comment. To avoid blocking follow-up PRs, I will address your comment in a follow-up.